packages/paste-core/components/side-modal/src/SideModalHeading.tsx

```diff
@@ -0,0 +1,29 @@
+import * as React from 'react';
+import * as PropTypes from 'prop-types';
+import {Heading} from '@twilio-paste/heading';
+import type {HeadingProps} from '@twilio-paste/heading';
+import type {BoxElementProps} from '@twilio-paste/box';
+
+export interface SideModalHeadingProps extends Omit<HeadingProps, 'marginBottom' | 'variant' | 'as'> {
+  children: NonNullable<React.ReactNode>;
+  as?: 'h1' | 'h2' | 'h3' | 'h4' | 'h5' | 'h6';
```

Comment:
praise: good continuation of API design from other, similar components. Build familiarity with using the system